tools/Messages

```diff
@@ -99,6 +99,15 @@ Usage()
     echo "       - \"list-indexed\" - list indexed projects"
     echo "       - \"get-repos\" - get list of repositories in the form of relative paths to source root for given project(s)"
     echo "       - \"get-repos-type\" - get repository type(s) for given project(s)"
+    echo "       - \"set\" - sets a particular project property (only primitive types)"
+    echo "                   in the webapp."
+    echo "                      Examples:"
+    echo "                          tabSize = 8"
+    echo "                          handleRenamedFiles = true"
```

Comment:
and also I think there is a wiki page listing the tunables so it should be also added there

Reply:
already there: https://github.com/oracle/opengrok/wiki/Per-project-configuration